requirements.txt

```diff
@@ -13,10 +13,10 @@ xmltodict==0.12.0
 lxml==4.6.2
 pytz==2021.1
 tzlocal==2.1
-libvirt-python==5.3.0; sys_platform != 'win32'
+libvirt-python==7.1.0; sys_platform != 'win32'
```

Comment:
What is the reason, this hasn't been upgraded? Are there any major changes that would break the code?

Reply:
The possibility of major changes put me off from upgrading it. It's possible that it'd just work, it's just as well possible that it breaks everything :p

Reply:
I'd welcome improvements at any point on this. If you can research if the 7.1.0 would run fine, we can leave it as is, otherwise your suggestion to split off is a nice solution too. I won't be upgrading the installed requirements on the platform so that won't be affected.